wled00/json.cpp

```diff
@@ -847,7 +847,7 @@ void serializeInfo(JsonObject root)
     #if defined(BOARD_HAS_PSRAM)
     root[F("psram")] = ESP.getFreePsram(); // Free PSRAM in bytes (backward compatibility)
     #endif
-    root[F("psramSize")] = ESP.getPsramSize() / (1024 * 1024); // Total PSRAM size in MB
+    root[F("psramSize")] = ESP.getPsramSize() / (1024UL * 1024UL); // Total PSRAM size in MB
   }
```

Comment:
Actually there is no overflow here - 1024*1024 requires 20bits, so int with 32bit would be sufficient for the calculation, not necessary to force everything into unsigned long. On the other side, this code is not time critical so unsigned lo g won't hurt either.